src/types.ts

```diff
@@ -59,7 +65,7 @@ export interface ContractConstantCallOptions extends Tx {
 }
 
 export interface AccountOperationOptions {
-  usePayableProxy?: boolean;
+  proxy?: ProxyType;
```

Comment:
picking up the conversation on breaking changes there. It's almost weirder to have it not be a breaking change for javascript but be a breaking change for typescript. How about we just have both `proxy` and `usePayableProxy` (you can put a deprecated comment on it if you want)